src/ctree-element-screen/ctree-segment-container.html

```diff
@@ -187,13 +187,15 @@
 
       previous() {
         if (this._variationIndex <= 0) return;
-
+        var component = this.shadowRoot.querySelector('#component');
```

Comment:
Hmm, I know this is done all over in this class (and I'm the one who did it), but it's really a lot of copy-paste code, which always is a good sign of an opportunity for improvement.  It's up to you, what you do (if anything), but a simple improvement would be to make a simple function to get the component.  A more sophisticated improvement would be to do something consistent with Polymer and see if we can set `this.$.component` when we create the component ID in `_segmentChanged`, which would prevent us from needing to use the more complicated (and inconsistent) method of calling querySelector.  That said, you'd need to test that carefully, because I'm not sure if there would be any negative implication of doing that (or maybe we don't even need to set it, maybe `this.$` automatically would work once the ID is set).